src/ckeditor-signature/InsertSignatureCommand.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @author Daniel Kesselberg <mail@danielkesselberg.de>
+ *
+ * Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command'
```

Comment:
do we have to install `@ckeditor/ckeditor5-core` explicitly?

Reply:
I don't think so. `@ckeditor/ckeditor5-editor-balloon` has `@ckeditor/ckeditor5-core` as dependency. But eslint does complain about it.

Reply:
I think eslint is not wrong. npm resolves a deep hierarchy of dependencies, not a flat one. we should depend on any import explicitly.

Reply:
Added @ckeditor/ckeditor5-core as dependency